bin/MadGraph5_aMCatNLO/cards/production/13TeV/exo_diboson/Spin-2/RSGraviton_hh_hVVhbb_fullLep/RSGraviton_hh_hVVhbb_fullLep_narrow_M1000/RSGraviton_hh_hVVhbb_fullLep_narrow_M1000_proc_card.dat

```diff
@@ -0,0 +1,10 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set gauge unitary
+set complex_mass_scheme False
+import model RS_exovv
+define lep = e+ e- mu+ mu-
```

Comment:
Are you sure you don't want tau's here?  Most "to LL" samples include taus because of the leptonic decay contribution